packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -51,13 +51,15 @@
                 v-oc-tooltip="showSpaceMemberLabel"
                 :aria-label="showSpaceMemberLabel"
                 appearance="raw"
-                @click="openSidebarSharePanel(resource)"
+                @click="openSidebarSharePanel(resource as SpaceResource)"
```

Comment:
Would be nice to figure out how to type slots :-D

Reply:
Turns out IntelliJ is smart enough to deduce the type from the slot declaration, vue-tsc apparently isn't... anyhow this is the correct fix, as resource-tiles component  just deals with resources and that's why we need to downcast here. 

Would be great if Vue had support for generic components 👀

Reply:
haha +1 for generic components 😅